Foundation/FileManager.swift

```diff
@@ -343,6 +343,9 @@ open class FileManager : NSObject {
         This method replaces fileSystemAttributesAtPath:.
      */
     open func attributesOfFileSystem(forPath path: String) throws -> [FileAttributeKey : Any] {
+#if os(Android)
+        NSUnimplemented()
```

Comment:
In this line, `NSUnimplemented()` means 'This hasn't been implemented yet, but it will be in the future' - whereas `NSUnsupported()` means that 'This will not be implemented on this platform'. Can you confirm this is intended to be implemented in the future? If not, using `NSUnsupported()` would be preferable.

Reply:
I think we should keep it NSUnimplremented, as someone later should be able to implement it in Android.

Reply:
OK